src/rocketData.py

```diff
@@ -14,108 +47,77 @@ class RocketData:
 
     Methods
     -------
-    all_rocketdata
+    all_rocket_data()
         Description: returns an array of all sensor data to be used in convert_to_csv_string
 
-    convert_to_csv_string
+    convert_to_csv_string()
         Description: converts all of the data into a csv string
 
-    send_to_airbrakes
+    send_to_airbrake()
         Description: sends an object that contains altitude, velocity, and acceleration
         info to calculations, which will send it to airbrakes (idk if this is how it works
         but send_to_airbrakes will send an object with relevant info wherever it has to go)
 
-    send_to_blackbox
+    send_to_black_box()
         Description: converts data to a csv string to be sent to blackbox
 
-    send_to_ground
+    send_to_ground()
         Description: takes critical data (what exactly?) and converts it to a string to be sent to the groundstation
 
     """
-    data = {}
-
     _lsm = None
     _bme = None
     _adx = None
+    _strain_gauges = None
+    _velocity = 0
 
     def __init__(self):
         self._bme = Bme()
         self._lsm = Lsm()
         self._adx = Adx()
-
-        self.data = {
-            'bme': {
-                'temperature': self._bme.temperature(),
-                'pressure': self._bme.pressure(),
-                'humidity': self._bme.humidity(),
-                'altitude': self._bme.altitude(),
-            },
-            'lsm': {
-                'temperature': self._lsm.temperature(),
-                'acceleration': self._lsm.acceleration(),
-                'gyroscope': self._lsm.gyroscope(),
-                'magnetometer': self._lsm.magnetometer(),
-            },
-            'adx': {
-                'acceleration': self._adx.acceleration(),
-            },
-            'strain_gauges': [
-                float,
-                float,
-                float,
-                float,
-                float,
-                float,
-                float,
-                float,
-                float,
-                float,
-                float,
-                float,
-            ],
-            'timestamp': float
-        }
-
-
-    @property
-    def data(self):
-        return self.data
-
-
-    @property
-    def timestamp(self):
-        return self.data['timestamp']
-
-
-    @timestamp.setter
-    def timestamp(self, ts):
-        self.data['timestamp'] = ts
-
-
-    # Set all sensor data -> get rid of this or make it an update all values function? shouldn't need to manually set data
-    @data.setter
-    def set_data(self, vals):
-        pass
-
-    def all_rocketdata(self):
+        self._strain_gauges = Hx711()
+        self._velocity = 0
+        self.timestamp = time.time()
+
+
+    def refresh(self):
+        previous_altitude = self._bme.altitude
+        previous_timestamp = self.timestamp
+
+        self._bme.refresh()
+        self._lsm.refresh()
+        self._adx.refresh()
+        # TODO: Make HX711s refresh function
```

Comment:
calling refresh will not update RocketData, you need to update rocketdata by refreshing then accessing the values and updating manually